dash/App/Notify/index.js

```diff
@@ -364,7 +364,7 @@ class Notify extends React.Component {
             <div
               className='notifyInputOption notifyInputProceed' onMouseDown={() => {
                 link.send('tray:openExternal', url)
-                link.send('tray:action', 'backDash')
+                link.send('tray:action', 'closeDash')
```

Comment:
let's not change this functionality as this PR it just about adding electronegativity and we should try to minimize regression. in addition, there are many calls to the `openExternal` IPC function that used to close the dash that now don't.

Reply:
ok

Reply:
Why is there a backDash call here when we are closing it though?

Reply:
yeah, not sure what the intention was in this case. @floating?